src/main/java/seedu/address/model/item/ItemList.java

```diff
@@ -83,4 +83,12 @@ protected ItemList find(String searchString, ItemList il) {
         }
         return il;
     }
+
+    public int indexOf(Item item) {
+        return list.indexOf(item);
+    }
+
+    public Item set(int index, Item item) {
```

Comment:
I feel that the name set is a little ambiguous, maybe you can consider changing it to replaceItem?

Reply:
ok!

Reply:
ohh I saw wrongly, my bad